doc/releases/changelog-0.29.0.md

```diff
@@ -111,7 +111,10 @@
   ```
 
   This operation can be differentiated with hardware-compatible methods like parameter shifts
-  and it supports parameter broadcasting/batching, but not both at the same time.
+  and it supports parameter broadcasting/batching, but not both at the same time. Learn more by
```

Comment:
Is it no the same, just we have two different implementations?

Reply:
I'll have to defer to @dwierichs for an answer!

Reply:
I'm not sure I understand the question right, but the support table is
|                         | scalar parameters | broadcasted parameters |
|-------------------------|-------------------|------------------------|
| auto-differentiable     |         yes          |         yes               |
| hardware-differentiable |         yes          |             no           |